### ISSUE TITLE
MySQL database connection might require charset

### ISSUE THREAD

**Issue by jerry1970** [devvoh/parable#19]
After searching for the reason why the `json_encode` failed that I mentioned in #18 , I found out it had to do with the encoding of the data. I solved this by specifying the charset to the DSN.

In `ORM\Database`, method `createPDOMySQL` I changed added *`;charset=utf8`* to the DSN:
```
// old
$dsn = 'mysql:host=' . $location . ';dbname=' . $database;
// new
$dsn = 'mysql:host=' . $location . ';dbname=' . $database . ';charset=utf8';
```
I now get properly formed data from the database and the `json_encode` succeeds.

Since I used a fixed charset in my previous project and it worked right away, I never spent time figuring out why the PDO connection requires me setting the charset.

With PSR requiring everything in UTF8 and almost everybody creating new projects in UTF8 I can only assume (silly me) that people set their data fields to UTF8 as well so for new projects I don't think adding the fixed charset to the DSN is bad. But for a framework default I can imagine you would want to turn that into a variable.

**Comment by jerry1970** [comment]
Made Pull Request #20 for a suggestion.

**Comment by devvoh** [comment]
Fixed and merged through PR #20.

Will be merged back into master and released in a few days.
